extensions/amp-render/1.0/storybook/Basic.amp.js

```diff
@@ -107,3 +107,48 @@ export const WithBindableSrc = () => {
 WithBindableSrc.story = {
   name: 'With bindable src',
 };
+
+export const WithAmpScriptSrc = () => {
+  return (
+    <>
+      <script
+        async
+        custom-element="amp-script"
+        src="https://cdn.ampproject.org/v0/amp-script-0.1.js"
+      ></script>
```

Comment:
Should this script be part of `extensions` in parameters instead?

https://github.com/ampproject/amphtml/blob/2e33c79f0e3e5a551ccf7a18d8e7ac58179f8735/extensions/amp-render/1.0/storybook/Basic.amp.js#L28-L30

Reply:
You're right, that worked.